Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1759,6 +1759,26 @@ class BrowserViewController: UIViewController {
         duckDuckGoPopup = popup
         popup.showWithType(showType: .flyUp)
     }
+    
+    func showBookmarkController() {
```

Comment:
can be private

Reply:
cant cause this is being used inside KeyCommands extension